### DISCUSSION TITLE
volcalc temperature selection

### DISCUSSION THREAD

**Discussion by marwashahin98** [Meredith-Lab/volcalc#110] [Q&A]
Hello,

As I understand, when calculating the volatility values using the "calc_vol" function, the model estimates the temperature to be at 20 degrees Celsius by default. I would like to calculate volatility at a different temperature, 5 degrees for example. How could this be achieved? 

Thanks for your feedback!

Marwa

**Comment by Aariq** [comment]
Hi Marwa,

Unfortunately it will involve some extensive changes to how `volcalc` works.  We hope to implement this, but will need time.  You could "subscribe" to the open issue we have for this to get updates: https://github.com/Meredith-Lab/volcalc/issues/41
